modules/checklists/assets/js/meta-box.js

```diff
@@ -870,11 +870,11 @@
 
   // Exposes and initialize the object
   window.PP_Checklists = PP_Checklists;
-  // RankMath plugin breaks if we
-  if (typeof rankMath !== 'undefined') {
+  // RankMath and Yoast SEO plugins break if we init immediately
+  if (typeof rankMath !== 'undefined' || typeof YoastSEO !== 'undefined') {
     setTimeout(function () {
       PP_Checklists.init();
-    }, 2500);
+    }, 3000);
```

Comment:
i just wondering, please help test if the site has rankMath and yoast at the same time. is it solve the issue or still breaking. because if each plugin stand alone i think it will be fine. if my opinion valid i think need separate block code for rankMath and yoast.

Reply:
You are correct, if both plugins activated at the same time, it is breaking. 

I add another validation if both activated.